.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @clerk/docs
```

Comment:
@alexisintech ever notice these little 🔴  red circles in your diff view? github expects a new line at the end of many file types and thats why it's showing that. you can add this to your vscode settings and it'll just do it onSave for ya without ever having to think about it and then poof, no more annoying 🔴 circles on yo PRs... 

https://stackoverflow.com/questions/44704968/visual-studio-code-insert-newline-at-the-end-of-files

Reply:
Ive brought this up before, there needs to be a Prettier config or .editorconfig that handles this for everyone.

Reply:
woah fireee.
and yeah @JacobMGEvans adding prettier is something we want to add, but when i previously added it, it was messing up our custom components and i was really busy and didn't have time to dig and figure out how to get it to ignore our custom components so i just stripped prettier for the time being